.github/workflows/ci.yml

```diff
@@ -16,7 +16,10 @@ env:
 
 jobs:
   engine-tests:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
intended?

Reply:
Yeah, this is required in order to run the device emulator on the cloud (requires hardware acceleration)